tests/tests/test_mtls.py

```diff
@@ -364,9 +326,7 @@ def test_mtls_enterprise_without_client_cert(self, setup_ent_mtls):
             "pending", max_wait=device_not_present_timeout_seconds * 0.5, no_assert=True
         ):
             devauth.decommission(device["id"])
-        setup_ent_mtls.device.run(
-            "systemctl start %s" % setup_ent_mtls.device.get_client_service_name()
-        )
+        setup_ent_mtls.device.run("systemctl start mender-authd")
```

Comment:
if we have `get_client_service_name` should we have also something for `mender-authd`?

Reply:
We had `get_client_service_name` because at the time, it could be two different ones. Not anymore :)